import/WebView.qml

```diff
@@ -12,6 +12,7 @@
 import QtQuick 2.0
 import Sailfish.Silica 1.0
 import Sailfish.WebView 1.0
+import Sailfish.WebView.Components 1.0
```

Comment:
While these are components... Some how this feels that it's given too much space. All of them are dialogs but not necessary in future. Alert, Confirm, and Prompt are JS popups (input related) and input/select handlers are also input related. Hmm, @chriadam how does something like ```Sailfish.WebView.Input``` feels to you? ```Input``` is also a bit overloaded but maybe we could come up with something else from this.